app/src/command.rs

```diff
@@ -128,9 +128,11 @@ impl<'app, D: Dependencies<'app>> Application<'app, D> {
             })?;
 
         #[cfg(feature = "op-upgrade")]
-        if let Some(params) = &attributes.eip1559_params {
-            self.gas_fee.set_parameters_from_attrs(params);
-        }
+        self.gas_fee
+            .set_parameters_from_extra_data(parent.block.header.extra_data).map_err(|e| {
```

Comment:
This is kind of a weird definition because it means the fee is constant (inductively: the current block is equal to its parent, which is equal to its parents, etc.). But if it works for the integration test then it is probably closer to correct than what we currently have.